pkg/apiserver/statement/queries.go

```diff
@@ -214,10 +235,19 @@ func QueryPlans(
 
 func QueryPlanDetail(
 	db *gorm.DB,
+	cacheService *sysschema.CacheService,
 	beginTime, endTime int,
 	schemaName, digest string,
-	plans []string) (result Model, err error) {
-	fields := getAllAggrFields()
+	plans []string,
+) (result Model, err error) {
+	aService, err := newAggrService(db, cacheService)
+	if err != nil {
+		return
+	}
+	fields, err := aService.getAggrs()
```

Comment:
It seems to be confusing to be :  `fields, .. = xxx.getAggrs() `

You may either:

- `fields, .. = xxx.getFields()`
- Or `aggrs, .. = xxx.getAggrs()`  // But what is Aggrs? I think this name is confusing.